src/sentry/static/sentry/app/views/stream.jsx

```diff
@@ -126,7 +129,7 @@ const Stream = createReactClass({
   },
 
   shouldComponentUpdate(nextProps, nextState) {
-    return !_.isEqual(this.state, nextState, true);
+    return !_.isEqual(this.state, nextState);
```

Comment:
there are some other usages of isEqual with three args in sentry

Reply:
https://github.com/lodash/lodash/blob/912d6b04a1f6b732508a6da72a95ec4f96bda154/lodash.js#L11545

I think this is proof the `true` is not needed

Reply:
Why ignore props changing?

Reply:
We only ignore if they haven't actually changed (using deep equality) to prevent the extra API call

Reply:
n/m I didn't see that this was previously there

Reply:
Our own implementation of `valueIsEqual` takes 3 args https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/utils.jsx#L37

Reply:
Maybe that explains the confusion